ovh/data_dedicated_installation_template_test.go

```diff
@@ -0,0 +1,43 @@
+package ovh
+
+import (
+	"regexp"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+)
+
+func TestAccDedicatedInstallationTemplateDataSource_basic(t *testing.T) {
+	testAccInstallationTemplateDatasourceConfig_404 := `data "ovh_dedicated_installation_template" "notemplate" {
+		template_name = "42"
+	}`
+	testAccInstallationTemplateDatasourceConfig_Basic := `data "ovh_dedicated_installation_template" "template" {
+		template_name= "debian12_64"
+	}`
+	config := testAccInstallationTemplateDatasourceConfig_Basic
+	resource.Test(t, resource.TestCase{
+		PreCheck:  func() { testAccPreCheckCredentials(t) },
+		Providers: testAccProviders,
+		Steps: []resource.TestStep{
+			{
+				Config: config,
```

Comment:
```suggestion
				Config: testAccInstallationTemplateDatasourceConfig_Basic,
```

and you can remove variable `config`